.github/workflows/test.yml

```diff
@@ -25,13 +25,13 @@ jobs:
       - name: Install python dependencies
         run: |
             pip install --upgrade pip
-            pip install -r ./validate/requirements.txt
+            pip install -r ./assert-tables/requirements.txt
       - name: Add tables to Deephaven
         run: python ./test/add-tables.py
       - name: Assert tables
         uses: ./.
         with:
           docker-compose-project-name: test-tables
```

Comment:
Is this still needed?  I think no.

Reply:
It is because `python ./test/add-tables.py` needs the same dependencies, and that needs to run before we test the base action. We can clear this up by copying the `requirements.txt` file to the `test` directory, that way it doesn't look like we're duplicating the same install

Reply:
Actually I thought about this, I think this is the way we should do this

```
      - name: Add tables to Deephaven
        run: |
          pip install --upgrade pip
          pip install -r ./assert-tables/requirements.txt #or ./test/requirements.txt
          python ./test/add-tables.py
```